public/app/percona/check/components/FailedChecksTab/FailedChecksTab.tsx

```diff
@@ -58,6 +78,19 @@ export const FailedChecksTab: FC = () => {
     setShowSilenced((currentValue) => !currentValue);
   };
 
+  const getRowProps = (row: Row<FailedCheckSummary>): TableRowProps => ({
+    key: row.original.serviceId,
+    className: styles.row,
+    // @ts-ignore
```

Comment:
Why is it needed?

Reply:
`onClick` wasn't recognized by the typings. I re-declared them to fix this.